Fixing FocusManager export and adding FocusManager testpage

samuelfreiberg opened the pull request:
The FocusManager export was exporting the class itself rather than an object with the FocusManager property. This PR fixes the export and adds a test file for the FocusManager API.

This will be backported to 62, 63, and 64

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/7020)

acoates-ms commented:
This looks like it'll break FocusManager.  The export from index.win32.js is to return the whole file for the FocusManager.get().  But now its being returned as a property on the module.  

The mismatch was only in the 0.62 branch.

NickGerleman commented on packages/react-native-win32-tester/src/js/examples-win32/APIs/FocusManagerAPI.tsx:
```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { FocusManager } from '../../../../../react-native-win32/Libraries/Utilities/FocusManager.win32';
```
For 0.64+, we will want to import from “@office-iss/react-native-win32”.

Relative import should be for 0.62/0.63 where this was inside the same package, and importing the package from inside the package caused a circular dependency.

NickGerleman commented on packages/react-native-win32-tester/src/js/examples-win32/APIs/FocusManagerAPI.tsx:
```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
```
Missing msft copyright header with “@format” annotation.

samuelfreiberg commented:
> This looks like it'll break FocusManager. The export from index.win32.js is to return the whole file for the FocusManager.get(). But now its being returned as a property on the module.
> 
> The mismatch was only in the 0.62 branch.

That's what I thought, but I went to test FocusManager here (I added test page locally) and it returned an error in the test app:
"Unable to get property "focus" of undefined or null reference."

I changed the export to { FocusManager } and that solved the issue. Thoughts?
